packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -191,20 +191,22 @@ describe('App: Settings', () => {
       // preferred editor selected from dropdown should have been persisted
       cy.__incorrectlyVisitAppWithIntercept()
       cy.get('[href="#/settings"]').click()
-      cy.wait(100)
+      cy.wait(200)
```

Comment:
Why do we need `cy.wait` here? Shouldn't it just wait until the Device Settings is rendered automatically?

Reply:
I wondered that too, but it's not, I guess. I changed it the wait time because it was failing for me locally.

Reply:
I see. I wonder if we can eliminate the need for wait at all? It shouldn't be necessary here.

Reply:
FWIW when I run these locally, the wait isn't necessary for this spec to pass. A lot of these waits date back a few months and predate a lot of lifecycle enhancements, but if you're still seeing failures then something is off.